CONTRIBUTING.md

```diff
@@ -10,6 +10,21 @@ using [Azure/autorest.go][] and [Azure/autorest][]. These generated packages dep
 
 For bugs or feature requests you can submit them using the [Github issues page][issues] and filling the appropriate template.
 
+## Building
+
+SDKs are either "track 1" or "track 2":
```

Comment:
We might want to check with the PM team I believe the terms track 1/2 are internal terms and we try to avoid putting them on the repos.

Reply:
Is there a better term? This is in CONTRIBUTING.md.

Reply:
@elraikhm @czubair can you provide guidance on best terminology when documenting how the new and old builds

Reply:
Marketing terms are "new" for track2 and "old" for track 1. I would phrase it as 
There are old (track1) and new (track2) SDK versions in the repo, Old SDKs are found in the services/ and profiles/top-level folders. New SDKs are found in the sdk/ top-level folder.